### DISCUSSION TITLE
A Perpetual "Breaking Changes" log in the admin area

### DISCUSSION THREAD

**Discussion by MattWilcox** [craftcms/cms#15817] [Ideas]
It doesn't happen often, but sometimes there are breaking changes or required edits that have to be done in templates or config files as part of some core or plugin update.

Those are usually mentioned in the update notes, but are also easy to miss if bulk updating, like when there has been a substantial time between running updates resulting in many being listed.

It'd be handy if bullet-items for breaking changes (or if that's too granular then the actual update note as a whole) were persistent somewhere in the CMS, so they are much less likely to be missed.

I'm imagining a "Breaking changes" log that shows each one that any particular project has had, and that displays as a notification count until a dev goes in and ticks a "yup, seen and delt with" for each such issue.

This way it'd be harder to have breaking changes go un-noticed.

**Comment by brandonkelly** [comment]
Related, Craft 4.13 and 5.5 are [adding](https://github.com/craftcms/cms/pull/15829) a `--minor-only` option to the `update` command, making it easier to update Craft and plugins without introducing any breaking changes (assuming everything is being good about semantic versioning).
